src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -643,4 +643,37 @@
   <data name="PX1072Title_CreateSharedGraphVariable" xml:space="preserve">
     <value>Creating a graph for a BQL query is a resource consuming operation. To decrease the number of these operations, create a variable with a shared graph that can be reused by BQL queries.</value>
   </data>
+  <data name="PX1062MessageFormat" xml:space="preserve">
```

Comment:
@EAndrosova  please review the diagnostic message template: `Graphs and graph extensions cannot declare static {0}`
The list of possible format arguments (they are also here in resources):
- mutable fields
- mutable properties
- views
- actions

Reply:
@SENya1990  I think we need an object in this sentence. Can we change it to "You cannot declare static {0} in graphs or graph exntesions."

Reply:
Yes, I'll change

Reply:
done